process-automation/latest/model/forms.md

```diff
@@ -186,6 +186,32 @@ The advanced properties for a REST dropdown field are:
 | Path to array in JSON response | The path to the JSON response. Enter `.` to use the full path. |
 | ID property | The ID of the REST service. |
 | Label property | The name of the REST service. |
+| Conditional | Turn this option on if you would like to link your drop down list with another drop down list and to create a conditional relationship between them. For example, if you select a country from one drop down list, the second drop down list will only show cities from that country. |
+| Depends on | Select which other dropdown list you would like to connect with. |
+| If equal | Select which child entry of the **Depends on** field you want to work with and add subordinate entries for it. |
+
+To create a conditional relationship between two dropdown fields using Country and City as an example:
+
+1. Add two dropdown list elements to your form.
+
+2. Name one of them Country and the other one City.
+
+3. Select the Country dropdown list element and click the **Advanced** tab.
+
+4. Select **Manual** and turn on the **Conditional** field.
+
+5. Select City from the **Depends on** dropdown list.
+
+6. Select the City dropdown list element and click the **Advanced** tab.
+
+7. Add a name for the label and then add all the Countries using the **Add option** field.
+
+8. Select the Country dropdown list element again and click the **Advanced** tab.
+
+9. From the **If equal** drop down list select the Country you want to work with and then add all the cities you would like available in the drop down list.
+
+When using **REST Service** you can use the ID of the linked widget in the REST URL. For example, if your URL is `https://mydomain.com/get-cities/country=${Country}` the value inside `${}` is the ID of the linked widget. If my widget had an ID called `my-dropdown` your URL would be `https://mydomain.com/get-cities/country=${my-dropdown}`.
+The `${my-dropdown-id}` can be used in any position of the URL, for example you can also use `https://mydomain.com/country=${Country}/get-cities`. The result of this configuration is that your app knows the two dropdown lists are linked which means when you select UK from the country list it understands the replacement and calls the correct URL, which in this example would be `https://mydomain.com/get-cities/country=UK`.
```

Comment:
Can we remove the part `The result of this configuration is that your app knows the two dropdown lists............` till the end? This is the modeling part of the linked dropdown fields and it will be better not to reference what happens during runtime

Reply:
Done